src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1240 @@
+    # exclusion zone 'Yuma Proving Ground' frequency range 'F2'.
+    grant_g3_a['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3575000000,
+        'highFrequency': 3585000000
+    }
+
+    grant_g1_b = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the exclusion zone
+    # 'Yakima Firing Center' frequency range 'F1'.
+    grant_g1_b['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3650000000,
+        'highFrequency': 3660000000
+    }
+    grant_g2_b = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the exclusion zone
+    # 'Yakima Firing Center' frequency range 'F1'.
+    grant_g2_b['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3675000000,
+        'highFrequency': 3685000000
+    }
+    grant_g3_b = json.load(
+                open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    # Set the grant frequency to overlap with the modified
+    # exclusion zone 'Yakima Firing Center' frequency range 'F2'.
+    grant_g3_b['operationParam']['operationFrequencyRange'] = {
+        'lowFrequency': 3575000000,
+        'highFrequency': 3585000000
+    }
+
+    # Update the location 'X' of CBSD devices contained witin the exclusion zones
+    # 'Yuma Proving Ground'
+    device_a['installationParam']['latitude'] = 33.01805
+    device_a['installationParam']['longitude'] = -114.25253
+    # 'Yakima Firing Center'
+    device_b['installationParam']['latitude'] = 46.67553
+    device_b['installationParam']['longitude'] = -120.43033
+
+    # Creating conditionals for Cat B devices
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionals_b = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['installationParam']
+    del device_b['measCapability']
+
+    conditionals = [conditionals_b]
+
+    # Create the actual config.
+    config = {
+        'registrationRequests': [device_a, device_b],
+        'grantRequests': [
+            [grant_g1_a, grant_g1_b],
+            [grant_g2_a, grant_g2_b],
+            [grant_g3_a, grant_g3_b]],
+        'conditionalRegistrationData': conditionals
+        # TODO
+        # Need to add data base configurations.
+      }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_FDB_1_default_config)
+  def test_WINNF_FT_S_FDB_1(self, config_filename):
+    """Exclusion Zone Database Update"""
+
+    # Load the configuration file
+    config = loadConfig(config_filename)
+
```

Comment:
Do some light checking?

Reply:
FDB.1        Checking no. of grant requests (grant size).     # Very light checking of the config file.     self.assertEqual(len(config['grantRequests']), 3)